src/sites/babuk.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+import logging
+
+from bs4 import BeautifulSoup
+
+from db.models import Victim
+from net.proxy import Proxy
+from .sitecrawler import SiteCrawler
+
+
+class Babuk(SiteCrawler):
+    actor = "Babuk"
+
+    def scrape_victims(self):
+        with Proxy() as p:
+            r = p.get(f"{self.url}", headers=self.headers)
+
+            soup = BeautifulSoup(r.content.decode(), "html.parser")
+
+            logging.info(f"Got content: {soup is None}")
+            # get max page number
+            victim_list = soup.find_all("a", class_="leak-card p-3")
+
+            for victim in victim_list:
+                victim_name = victim.find("h5").text.strip()
+
+                published = victim.find("div", class_="col-auto published")
+                published_dt = datetime.strptime(
+                    published.text.strip(), "%Y-%m-%d %H:%M:%S")
+
+                if victim_name == "Hello world 1" or victim_name == "Mercy, journalists,chonky boi":
+                    # skipping news and updates
+                    continue
+
+                logging.info(
+                    f"Found victim: {victim_name} - Published date: {published_dt}")
+
+                victim_leak_site = self.url + victim.attrs["href"][1:]

```

Comment:
I assume the `[1:]` was to eliminate a leading `/` on the URL path. The URL in the `config.yaml` file shouldn't have a trailing slash, so please adjust this accordingly.